src/components/about-us/section-04/info-billboard.js

```diff
@@ -0,0 +1,138 @@
+import { CloseButton } from './map'
+import { colors } from '../constants/styles'
+import { content } from '../constants/data/section4-content'
+import { Location, LogoAndLink } from './info-card'
+import { screen } from '../utils/screen'
+import hyperlink from '../../../../static/asset/about-us/intl-co-link.png'
+import location from '../../../../static/asset/about-us/intl-co-location.png'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const MobileStyledBillboard = styled.div`
+  ${screen.desktopAbove`
+    display: none;
+  `}
+  background: #c7000a;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: ${props => !props.isOpened ? 'none' : 'block'};
+  z-index: 3;
```

Comment:
give a default `z-index` which is used for lower layer elements, and give this upper layer elements `z-index: ${defaultZIndex} + 1`